packs/equipment/ankhrav-duster.json

```diff
@@ -0,0 +1,89 @@
+{
+    "_id": "nymtBfivudt7t75q",
+    "img": "systems/pf2e/icons/default-icons/weapon.svg",
+    "name": "Ankhrav Duster",
+    "system": {
+        "baseItem": "knuckle-duster",
+        "bonus": {
+            "value": 0
+        },
+        "bonusDamage": {
+            "value": 0
+        },
+        "bulk": {
+            "value": 0.1
+        },
+        "category": "simple",
+        "containerId": null,
+        "damage": {
+            "damageType": "bludgeoning",
+            "dice": 1,
+            "die": "d4"
+        },
+        "description": {
+            "value": "<p>Acid drips from the ankhrav mandibles protruding from this +1 knuckle duster. Strikes with this weapon deal an additional @Damage[1[acid]] damage.</p>\n<p><strong>Activate—Caustic Jabs</strong> <span class=\"action-glyph\">1</span> (concentrate)</p>\n<p><strong>Frequency</strong> once per hour</p>\n<hr />\n<p><strong>Effect</strong> Until the end of your next turn, your Strikes with the ankhrav duster deal an additional 1d6 persistent acid damage.</p>\n<hr />\n<p><strong>Craft Requirements</strong> The initial raw materials must include the mandibles of an ankhrav.</p>"
+        },
+        "group": "brawling",
+        "hardness": 0,
+        "hp": {
+            "max": 0,
+            "value": 0
+        },
+        "level": {
+            "value": 4
+        },
+        "material": {
+            "grade": null,
+            "type": null
+        },
+        "price": {
+            "value": {
+                "gp": 90
+            }
+        },
+        "publication": {
+            "license": "ORC",
+            "remaster": true,
+            "title": "Pathfinder Howl of The Wild"
+        },
+        "quantity": 1,
+        "range": null,
+        "reload": {
+            "value": null
+        },
+        "rules": [],
```

Comment:
Consider automating additional acid damage:
```
{
	"damageType": "acid",
	"key": "FlatModifier",
	"selector": "{item|_id}-damage"
	"value": 1
}
```

Reply:
Can be done in a follow up, this will probably need a bit of cleanup once we're out of the data freeze.